CMSgov/mdct-mcr
# MDCT-2093 - Last Saved Time in Form Header

## Description
[MDCT-2093](https://qmacbis.atlassian.net/browse/MDCT-2093)

Adding text to the left of the `Leave Form` header button to indicate the last saved time of the current report.

For now design and product decided to only show the time for the current session (it won't show up until you save for the first time in that login, or after refreshing the page). In the future it should be easy to expand on this to show last saved from a previous session, by passing in the `report.lastAltered` time to the `getLocalHourMinuteTime` method.

### How to test
Create a report, enter it, and fill out any page. After saving it should display the accurate local time of when you saved.

## Code author checklist

## Reviewer checklist (two different people)
